setup.py

```diff
@@ -30,7 +30,20 @@
 
 import sys
 if sys.version_info[:2] < (3, 5):
-    raise RuntimeError("Python version >= 3.5 required.")
+    msg = ["This version of cocotb requires at least Python 3.5 to run."]
+    if sys.version_info[0] == 2:
+        msg += [
+            "If you have Python 3 installed on your machine try ",
+            "using 'pip3' instead of 'pip' to install cocotb."
+        ]
```

Comment:
Perhaps best here to recommend `python3 -m pip`, if `pip` was pointing at the wrong thing